### ISSUE TITLE
Always Showing OOPS Screen on webView in Android App or Emulator.

### ISSUE THREAD

**Issue by abhi-suman** [Paytm-Payments/Paytm_Android_App_Kit#26]
Hi Team, 

**It's Urgent**
I Just try to check this sample app and encountered the issue. 
![image](https://user-images.githubusercontent.com/33564965/32686841-0807162e-c6d4-11e7-98a3-f2d3cbe593a7.png)

This is going to come every time I run the application. 
Please help me out.

**Comment by amitabha2715** [comment]
I have faced this same issue..

**Comment by shubhanshusingh** [comment]
yes same applies for me, can't test and take it to production properly

**Comment by wangshanhai** [comment]
Me too

**Comment by rakeshgopathi** [comment]
Did anyone find a solution?

**Comment by shubhanshusingh** [comment]
@rakeshgopathi just using production service instead! same bug no solution, would suggest to use 2.0.0 version of SDK

**Comment by Wangab** [comment]
I'm using SDK 2.0.0.I have the same problem

**Comment by dpaksoni** [comment]
Same problem here. Using 2.0 version of SDK

**Comment by lalitappdeveloper** [comment]
same problem here.

**Comment by RozinaDarediya** [comment]
I am facing same issue..
Do anyone find solution ?

**Comment by Prabha10cs** [comment]
Same problem here. Using 2.0 version of SDK

**Comment by Wangab** [comment]
use sdk2.0. When you generate the CHECKSUMHASH, the parameters must be the same as the parameters in the request.

**Comment by sasidhar678** [comment]
You need to use unique ORDER_ID, CUST_ID, CALLBACK_URL (mostly provided by PayTM) and CHECKSUM for each and every request.
